python/lsst/ip/isr/ptcDataset.py

```diff
@@ -146,6 +146,11 @@ class PhotonTransferCurveDataset(IsrCalib):
          Dictionary keyed by amp names containing the integrated photocharge
          for linearity calibration.
 
+    Returns
```

Comment:
I'm very confused. This addition is wrong. It should not be there because we do not include returns sections in class documentation (it was removed from main). The "Version 1.1" text should be part of the above Notes section not a returns section.